pkg/defaults/defaults.go

```diff
@@ -408,4 +408,10 @@ const (
 
 	// EnableBPFBypassFIBLookup instructs Cilium to enable the FIB lookup bypass optimization for nodeport reverse NAT handling.
 	EnableBPFBypassFIBLookup = true
+
+	// WireguardSubnetV4 is a default wireguard tunnel subnet
+	WireguardSubnetV4 = "172.16.42.0/24"
```

Comment:
Is there something special about this range? I feel like I've hit issues recently because of two other applications both thinking this particular /24 is a great range to use for independent reasons. (Well I think ultimately the reasons come down to a combination of RFC1918 and Douglas Adams, but that's not *so* informative by itself..)

Reply:
Hehe :smile: I think I took it from the archlinux Wireguard example. I could change the 42 number to something less common, but ideally we should implement a subnet auto-detection for Wireguard.